react-carousel/test/plugins/infinite.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import setupCarousel from '../tools/setupCarousel';
+import { defaultOptions as infiniteDefaultOptions } from '../../src/plugins/infinite';
+
+describe('infinite', () => {
+  it('renders additional clones when in infinite mode', () => {
+    const sidesToClone = 2; // we create clones for the left and for the right side
```

Comment:
sides or slides?

Reply:
side as written in comment we create additional clones for the side of the carousel and for the right side

Reply:
ok